.travis.yml

```diff
@@ -4,7 +4,7 @@ dist: trusty
 addons:
   apt:
     sources:
-      - chef-stable-trusty
+      - chef-current-trusty
```

Comment:
I really like trying to test out an upcoming release. It's caught stuff recently.